code/game/objects/structures.dm

```diff
@@ -240,13 +240,17 @@
 
 //Damage
 /obj/structure/proc/take_damage(dam)
-	if(!dam || CHECK_BITFIELD(resistance_flags, INDESTRUCTIBLE|UNACIDABLE))
+	if(!dam || CHECK_BITFIELD(resistance_flags, INDESTRUCTIBLE))
```

Comment:
This should be done in a separate PR as the implications are kind huge.

Reply:
@Rohesie how important is this change?

Reply:
Same as above, this shouldn't be done in this PR.